sync/bug.py

```diff
@@ -376,10 +376,10 @@ def bug_ctx(self, bug_id):
         return MockBugContext(self, bug_id)
 
     def new(self, summary, comment, product, component, whiteboard=None, priority=None,
-            url=None):
+            url=None, bug_type="task"):
         self._log("Creating a bug in component %s :: %s\nSummary: %s\nComment: %s\n"
-                  "Whiteboard: %s\nPriority: %s URL: %s" %
-                  (product, component, summary, comment, whiteboard, priority, url))
+                  "Whiteboard: %s\nPriority: %s\nURL: %s\nType: %s" %
+                  (product, component, summary, comment, whiteboard, priority, url, bug_type))
```

Comment:
Here we can see how a long list of arguments start to make it difficult to read. 
probably in the future a dictionary and the use of a format with proper variable names would make it more readable than a long series of `%s`

```python
BUG_CREATION_LOG = """Creating a bug in component {product} :: {component}
Summary: {summary}
Comment: {comment}
Whiteboard: {whiteboard}
Priority: {priority}
URL: {url}
Type: {bug_type}
"""

# with a dictionary to store the data
bug_info = {'product': 'something', 'component': 'something_else', }

## python 2.7
self._log(BUG_CREATION_LOG.format(**bug_info))

## python 3.2+
self._log(BUG_CREATION_LOG.format_map(bug_info))
```